ebmbot/job_configs.py

```diff
@@ -290,6 +295,12 @@
                 "type": "schedule_job",
                 "job_type": "out_of_office_status",
             },
+            {
+                "command": "rota report",
+                "help": "Report who's next on tech support duty",
+                "type": "schedule_job",
+                "job_type": "rota_report",
+            },
```

Comment:
Do we trigger this with a `/remind` in Slack? What's your proposed schedule?

Reply:
Yes.  Every Monday and Friday at 9am?

Reply:
Fine